website/docs/security/authentication.md

```diff
@@ -84,8 +86,8 @@ If you are using the MongoDB Shell to connect to FerretDB, you can create a new
 
 ```js
 db.createUser({
-  user: 'newuser',
-  pwd: 'newpassword',
+  user: '<newuser>',
```

Comment:
Note that the vector search example uses double quotes, and the linter is configured to use them

Reply:
Noticed this too. Is this because the vector search docs is in `.mdx` format? The linter changes this back to single quotes as well.

Reply:
I see what's going on `--single-quote` option on `prettier` is making double quotes into single quotes.

https://github.com/FerretDB/FerretDB/blob/0cf6ca0344a6478290b659f09d5286450280d344/Taskfile.yml#L732-L733

Reply:
@Fashander what's the outcome of this?

Reply:
Not sure if the single quote is not in use or needed somewhere else. If I remove it, then all quotes will remain as-is.

Reply:
FYI, vector search examples are all in `*.js` files and currently double quote is enforced for `*.js` files. See below `fmt-js` doesn't have `--single-quote`, defaulting to enforcing double quote https://prettier.io/docs/rationale#strings. https://github.com/FerretDB/FerretDB/blob/0cf6ca0344a6478290b659f09d5286450280d344/Taskfile.yml#L707-L710